mountpoint-s3-csi-driver: update for version streams

shimmerjs opened the pull request:
Related: https://github.com/chainguard-dev/customer-issues/issues/2549

This PR updates the existing mountpoint-s3-csi-driver package to produce a mounpoint-s3-csi-driver-2.0 package in line with other version streams as used in the updated image definition.

shimmerjs committed:
mountpoint-s3-csi-driver: update for version streams

kranurag7 commented on mountpoint-s3-csi-driver-2.0.yaml:
```suggestion

```
Is there a reason to provide the package itself (this is main package) thinking....

shimmerjs replied:
Nope, just my ignorance. Forgot that package name is always provided

shimmerjs committed:
Update mountpoint-s3-csi-driver-2.0.yaml

Co-authored-by: kranurag7 <81210977+kranurag7@users.noreply.github.com>
Signed-off-by: alex weidner <shimmerjs@dpu.sh>

shimmerjs committed:
Merge branch 'main' into vs/mountpoint-s3-csi-driver

shimmerjs committed:
Merge branch 'main' into vs/mountpoint-s3-csi-driver

EyeCantCU committed:
chore(mountpoint-s3-csi-driver-2.0.yaml): Bump epoch to 2

We provide the non-streamed version of mountpoint-s3-csi-driver so we need to increment the epoch from 1 to 2

Signed-off-by: RJ Trujillo <rj.sampson@chainguard.dev>

EyeCantCU committed:
Merge branch 'main' into vs/mountpoint-s3-csi-driver
